src/app/domain/run.spec.ts

```diff
@@ -35,4 +35,21 @@ describe('Run', () => {
     expectRun('isScheduled', beforeStartTime, true);
     expectRun('isScheduled', betweenStartAndEndTimes, false);
   });
+
+  describe('sortByStartTimeDesc()', () => {
+    let runList: Run[];
+    const run2 = new Run({
+      startTime: endTime
+    });
+    const run3 = new Run({
+      startTime: afterEndTime
+    });
+    beforeEach(() => {
+      runList = [run2, run3, run];
+      runList.sort(sortByRunStartTimeDesc);
+    });
+    it('should sort runs by start date', async () => {
```

Comment:
Remove async and "should"?
```suggestion
    it('sorts runs by start date', () => {
```